andersen-homework/src/main/java/com/example/ticket/TicketApplication.java

```diff
@@ -1,26 +1,14 @@
 package com.example.ticket;
 
-import com.example.ticket.model.Ticket;
-import com.example.ticket.util.BusTicketValidator;
-import org.springframework.context.ApplicationContext;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import java.io.IOException;
-import java.util.List;
 
 
 @SpringBootApplication
```

Comment:
for 10 spring task use please just Spring, not SpringBoot :)

Reply:
okay teacher:) i have deleted this annotation

Reply:
And also remove dependency :)

Reply:
done